docs/resources/mpc_transcoding_template.md

```diff
@@ -0,0 +1,210 @@
+    preset               = 1
+    profile              = 4
+    ref_frames_count     = 4
+    height               = 0
+    width                = 0
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the transcoding template resource. If omitted,
+  the provider-level region will be used. Changing this creates a new resource.
+
+* `name` - (Required, String) Specifies the name of a transcoding template.
+
+* `common` - (Required, List) Specifies the common parameters. The [object](#common_object) structure is documented below.
+
+* `audio` - (Optional, List) Specifies the audio parameters. The [object](#audio_object) structure is documented below.
+
+* `video` - (Optional, List) Specifies the video parameters. The [object](#video_object) structure is documented below.
+
+<a name="common_object"></a>
+The `common` block supports:
+
+* `pack_type` - (Required, Int) Specifies the packaging type. Possible values are:
+  + **1**: HLS
+  + **2**: DASH
+  + **3**: HLS+DASH
+  + **4**: MP4
+  + **5**: MP3
+  + **6**: ADTS
+
+-> If pack_type is set to 5 or 6, do not set video parameters.
+
+* `pvc` - (Optional, Bool) Specifies Whether to enable low bitrate HD. The default value is false.
+
+* `hls_interval` - (Optional, Int) Specifies the HLS segment duration. This parameter is used only when `pack_type` is
+  set to 1 or 3. The value ranges from 2 to 10. The default value is 5.The unit is second.
+
+* `dash_interval` - (Optional, Int) Specifies the dash segment duration. This parameter is used only when `pack_type`
+  is set to 2 or 3. The value ranges from 2 to 10. The default value is 5. The unit is second.
+
+<a name="audio_object"></a>
+The `audio` block supports:
+
+* `codec` - (Required, Int) Specifies the audio codec. Possible values are:
+  + **1**: AAC
+  + **2**: HEAAC1
+  + **3**: HEAAC2
+  + **4**: MP3
+
+* `sample_rate` - (Required, Int) Specifies the audio sampling rate. Possible values are:
+  + **1**: AUDIO_SAMPLE_AUTO
+  + **2**: AUDIO_SAMPLE_22050 (22,050 Hz)
+  + **3**: AUDIO_SAMPLE_32000 (32,000 Hz)
+  + **4**: AUDIO_SAMPLE_44100 (44,100 Hz)
+  + **5**: AUDIO_SAMPLE_48000 (48,000 Hz)
+  + **6**: AUDIO_SAMPLE_96000 (96,000 Hz)
+
+* `channels` - (Required, Int) Specifies the number of audio channels. Possible values are:
+  + **1**: AUDIO_CHANNELS_1
+  + **2**: AUDIO_CHANNELS_2
+  + **6**: AUDIO_CHANNELS_5_1
+
+* `bitrate` - (Optional, Int) Specifies the audio bitrate. The value is 0 or ranges from 8 to 1,000.
+  The default value is 0. The unit is kbit/s.
+
+* `output_policy` - (Optional, String) Specifies the output policy. Possible values are **discard** and **transcode**.
+  The default value is transcode.
+
+<a name="video_object"></a>
+The `video` block supports:
+
+* `output_policy` - (Optional, String) Specifies the output policy. Possible values are **discard** and **transcode**.
+  The default value is transcode.
+
```

Comment:
pls add a note to show more details:
+ If output_policy is set to discard in video parameters and transcode in audio parameters, only audio is output.
+ If output_policy is set to transcode in video parameters and discard in audio parameters, only video is output.
+ output_policy cannot be set to discard in video and audio parameters at the same time.
+ If output_policy is set to transcode in video and video parameters at the same time, video and video are output.

Reply:
fixed